pkg/util/tracing/tracingservicepb/tracing_service.proto

```diff
@@ -15,14 +15,19 @@ option go_package = "tracingservicepb";
 import "gogoproto/gogo.proto";
 import "util/tracing/tracingpb/recorded_span.proto";
 
-message SpanRecordingRequest {
+// GetSpanRecordingsRequest requests a nodes' inflight span registry to return
```

Comment:
Move this comment to the RPC definition, it'll show up when jumping to interface/method, instead of having to jump to the request type. "GetSpanRecordings returns a nodes' ...", and refer to trace_id as TraceID instead.

Reply:
done.